src/components/Daos/DaosPage.tsx

```diff
@@ -48,17 +49,31 @@ class DaosPage extends React.Component<IProps, IState> {
     super(props);
 
     this.state = {
+      isMobile: window.innerWidth <= 550,
```

Comment:
wow this is starting to look like a big hack, definitely not a "tweak".  Can this not be done with css and @media rules?

Reply:
not this part no because its in the react code